recipes.js

```diff
@@ -1,11 +1,65 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const chalk = require("chalk");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
+
+const express = require("express");
```

Comment:
We are also not creating routes here, so we don't need to require express either !